.github/path-ci/run-smoke.ps1

```
@@ -81,49 +81,50 @@ if (-not $found) {
-$hadFountClick = Test-Path Env:FOUNT_CLICK
-$prevFountClick = $env:FOUNT_CLICK
-$env:FOUNT_DIR = $Root
-$env:FOUNT_CLICK = '1'
-try {
-	. $Fount open
-	if ($LastExitCode -ne 0) {
-		throw "FOUNT_CLICK open exited with $LastExitCode"
+if ($env:OS -eq 'Windows_NT') {
+	Write-Host '== wt start =='
+	$script:WindowsTerminalStartCaptured = $null
+	function Start-Process {
```

PSScriptAnalyzer (1.25.0)

[warning] Function 'Start-Process' has verb that could change system state. Therefore, the function has to support 'ShouldProcess'.

(PSUseShouldProcessForStateChangingFunctions)